### PR TITLE
Feature/kon 407 add cpr service from serviceplatformen

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: CPR/ServiceplatformenCprServiceResult.php [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+     * Get first name.
+     *
+     * @return string
+     */
+    public function getFirstName(): string
+    {
+        return $this->response
+            ->persondata
+            ->navn
+            ->fornavn;
```
Comment (rimi-itk) [inline CPR/ServiceplatformenCprServiceResult.php:40]: This makes a lot of assumptions of the structure/value of `$this->response`. Maybe we need some additional checks, e.g. using https://symfony.com/doc/current/components/property_access.html.

File: CPR/ServiceplatformenServiceFactory.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+     * @param string $serviceplatformenUserUuid
+     * @param string $personBaseDataExtendedServiceContract
+     * @param string $personBaseDataExtendedServiceEndpoint
+     * @param string $personBaseDataExtendedServiceUuid
+     *
+     * @return PersonBaseDataExtendedService
+     *
+     * @throws CprException
+     */
+    public static function createPersonBaseDataExtendedService(string $azureTenantId, string $azureApplicationId, string $azureClientSecret, string $azureKeyVaultName, string $azureKeyVaultSecret, string $azureKeyVaultSecretVersion, string $serviceplatformenServiceAgreementUuid, string $serviceplatformenUserSystemUuid, string $serviceplatformenUserUuid, string $personBaseDataExtendedServiceContract, string $personBaseDataExtendedServiceEndpoint, string $personBaseDataExtendedServiceUuid)
```
Comment (rimi-itk) [inline CPR/ServiceplatformenServiceFactory.php:46]: That's a lot of parameters! Consider adding line breaks to improve readability.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
A few comments have been added.

File: CPR/ServiceplatformenServiceFactory.php (hunk 2 above)
Comment (lats1) [reply CPR/ServiceplatformenServiceFactory.php:46]: I do agree, but our coding standard forces me to have them on the same line.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**